owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsPictureUploadsFragment.kt

```diff
@@ -117,6 +120,10 @@ class SettingsPictureUploadsFragment : PreferenceFragmentCompat() {
                 prefPictureUploadsPath?.summary = DisplayUtils.getPathWithoutLastSlash(it.uploadPath)
                 prefPictureUploadsSourcePath?.summary = DisplayUtils.getPathWithoutLastSlash(it.sourcePath.toUri().path)
                 prefPictureUploadsOnWifi?.isChecked = it.wifiOnly
+                prefPictureUploadsOnCharging?.apply {
+                    isVisible = it.wifiOnly
```

Comment:
When I implemented the auto upload pictures and videos sections, we had a quick meeting and we decided to keep items not **enabled** instead of not **visible** when they cannot be activated yet, so that user perceives the app's value and all its possibilities since the beginning and not by discovering them when they activate other options.

On the other hand, it might be a bit weird in this case, as everything is disabled when the switch button is off, and then when turning it on, this preference would continue being disabled.

What do you think about this? Should we keep the preference not visible or not enabled?